donkeycar/pipeline/types.py

```diff
@@ -0,0 +1,61 @@
+import os
+from typing import Any, List, Optional, cast
+
+from donkeycar.parts.tub_v2 import Tub
+from donkeycar.utils import load_image_arr, normalize_image
+from typing_extensions import TypedDict
+
+TubRecordDict = TypedDict(
+    'TubRecordDict',
+    {
+        'cam/image_array': str,
+        'user/angle': float,
+        'user/throttle': float,
+        'user/mode': str,
+        'imu/acl_x': Optional[float],
```

Comment:
We should use two 3-D arrays here, one for acceleration one for gyro.

Reply:
I am keeping the representation of the underlying dictionary here the same as what is actually stored.

Reply:
I am keeping it the same as what is actually stored.